api/src/main/java/jakarta/servlet/http/HttpServlet.java

```diff
@@ -394,6 +446,7 @@ protected void doOptions(HttpServletRequest req, HttpServletResponse resp) throw
         boolean ALLOW_GET = false;
         boolean ALLOW_HEAD = false;
         boolean ALLOW_POST = false;
+        boolean ALLOW_PATCH = false;
```

Comment:
You are missing the code to set this to 'true' if the method is present.

Reply:
Fixed.